packages/theme/src/action-menu/responsive.less

```
@@ -0,0 +1,21 @@
+@import '../custom.less';
+@import './vars.less';
+
+@action-menu-prefix-cls: ~'@{css-prefix}action-menu';
+@dropdown-prefix-cls: ~'@{css-prefix}dropdown';
+@dropdown-item-prefix-cls: ~'@{css-prefix}dropdown-item';
+
+@media screen and (max-width: 1280px) {
+  .@{action-menu-prefix-cls} {
```

Stylelint (17.4.0)

[error] Unknown word action-menu-prefix-cls (CssSyntaxError)

(CssSyntaxError)